.github/workflows/pr-to-update-winget.yml

```diff
@@ -0,0 +1,129 @@
+name: PR for updating winget
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  winget-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout winget-automation
+        uses: actions/checkout@v4
+        with:
+          ref: winget-automation
+          token: ${{ secrets.KINVOLK_REPOS_TOKEN }}
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          user=${{github.actor}}
+          if [ -z $user ]; then
+            user=vyncent-t
+          fi
+          git config --global user.name "$user"
+          git config --global user.email "$user@users.noreply.github.com"
+
+      - name: Create node.js environment
+        uses: actions/setup-node@v4
+        with:
+          node-version: "21"
+
+      - name: Set latest tag
+        run: |
+          echo "Setting latest tag"
+          latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          # Remove the 'v' from the tag
+          latestTag=${latestTag#v}
+          echo "LATEST_HEADLAMP_TAG=$latestTag" >> $GITHUB_ENV
+          echo $latestTag
+
+      - name: Install winget dependencies
+        run: |
+          cd $GITHUB_WORKSPACE/app/windows/winget
+          npm install
```

Comment:
We don't have dependencies in winget.

Reply:
ah, I have been pavlov'd from seeing the package.json so close to the working dir lol

![image](https://github.com/headlamp-k8s/headlamp/assets/78232183/cc3e57e9-29b0-49fd-9e85-a022c531381d)

Reply:
it seems we need this step so that the winget script can use node-fetch that the fetch = require('node-fetch') implements

this is error on removing npm install, but it works with it used

![image](https://github.com/headlamp-k8s/headlamp/assets/78232183/cb4e60b7-621f-4d5f-8582-3c06a54eeea8)